README.md

```diff
@@ -93,6 +93,50 @@ var r = await client.PostPointAsync(dbName, valMixed);
 
 A collection of points can be posted using `await client.PostPointsAsync (dbName, points)`, where `points` can be collection of different types of `InfluxDatapoint`
 
+#### Writing strongly typed data points to DB
+
+```Csharp
+class YourPoint
+{
+    [InfluxDbMeasurementName]
+    public string Measurement { get; set; }
+
+    [InfluxDbTime]
+    public DateTime Time { get; set; }
+
+    [InfluxDbPrecision]
+    public TimePrecision Precision { get; set; }
+
+    [InfluxDbRetentionPolicy]
+    public InfluxRetentionPolicy Retention { get; set; }
+
+    [InfluxDbField("FieldName")]
+    public string FieldProperty { get; set; }
+
+    [InfluxDbTag("TagName")]
+    public string TagProperty { get; set; }
+
+}
+
+var point = new YourPoint
+{
+    Time = DateTime.UtcNow,
+    Measurement = measurementName,
+    Precision = TimePrecision.Seconds,
+    FieldProperty = "FieldValue",
+    TagProperty = "TagValue",
+    Retention = new InfluxRetentionPolicy() { Name = "Test2" };
+};
+
+var r = await client.PostPointAsync(dbName, point);
```

Comment:
In this example where are you setting actual value of the fields?

Reply:
I have extended it with different typed fields. In the example I'm setting the values of the fields in the constructor for the `YourPoint` object, but obviously it can be done in any other way you would set the value of a field.